scipy/optimize/_highs/meson.build

```diff
@@ -7,7 +7,12 @@ highs_define_macros = [
   '-DHIGHS_VERSION_MAJOR=1',  # don't care about this, look at CMakelists.txt
   '-DHIGHS_VERSION_MINOR=0',
   '-DHIGHS_VERSION_PATCH=0',
-  '-DHIGHS_DIR=' + meson.current_source_dir() / '..' / '..' / '_lib' / 'highs'
+  '-DHIGHS_DIR=' + meson.current_source_dir() / '..' / '..' / '_lib' / 'highs',
+  '-UOPENMP',
```

Comment:
Figured it wouldn't hurt to have these everywhere and it's needed in multiple libraries now.  Please refactor into another list if desired

Reply:
This seems fine to me.